azext_iot/iothub/params.py

```diff
@@ -246,3 +248,70 @@ def load_iothub_arguments(self, _):
             options_list=["--content-type", "--ct"],
             help="MIME Type of file.",
         )
+
+    with self.argument_context("iot hub state") as context:
+        context.argument(
+            "filename",
+            options_list=["--filename", "-f"],
+            help="The path to the file where the state information will be stored."
+        )
+        context.argument(
+            "replace",
+            options_list=["--replace", "-r"],
+            help="If this flag is set, then the command will overwrite the state of the destination hub."
+        )
+        context.argument(
+            "overwrite_file",
+            options_list=["--overwrite-file", "--of"],
```

Comment:
In the in-flight PR for the prior implementation, we changed `--overwrite-file` to `--force`, and no alias is needed

Reply:
I will change the replace to force too

Reply:
We talked about aligning on `--replace`.